templates/release_checklist.md

```diff
@@ -0,0 +1,157 @@
+<!-- Checklist generated by release toolkit -->
+
+<!-- scheduled_release_only -->
+
+## Before the Release ({{before_release_date}})
+
+- [ ] Review the [release documentation](https://github.com/wordpress-mobile/release-toolkit-gutenberg-mobile/blob/develop/Releasing.md).
+- [ ] [Notifiy open Gutenberg Mobile PRs assigned to {{gb_mobile_version}} milestone]({{milestone_url}}).
+<details>
+<summary>Message Template</summary>
+<p>
+
+```markdown
+Hello!
+We will cut the {{gb_mobile_version}} release on {{release_date}}.
+I plan to circle back and bump this PR to the next milestone then,
+but please let me know if you’d rather us work to include this PR in {{gb_mobile_version}}.
+Thanks!
+```
+
+</p>
+</details>
+- [ ] Clone or update the [release toolkit](https://github.com/wordpress-mobile/release-toolkit-gutenberg-mobile)
+- [ ] Create WPiOS and WPAndroid builds from `develop`.
+- [ ] [Perform general writing flow tests on Android](https://github.com/wordpress-mobile/test-cases/tree/master/test-cases/gutenberg/writing-flow).
+- [ ] [Perform general writing flow tests on iOS](https://github.com/wordpress-mobile/test-cases/tree/master/test-cases/gutenberg/writing-flow).
+
+<!-- /scheduled_release_only -->
+
+## Create the Release ({{release_date}})
+
+- [ ] [Verify Aztec is up to date](https://github.com/wordpress-mobile/release-toolkit-gutenberg-mobile/blob/develop/Releasing.md#handling-aztec-updates).
+- [ ] Run `./release_automation.sh` from the release toolkit to generate the main apps integration PRs [\*](#).
+- [ ] Notify team that the release has been cut.
```

Comment:
I miss the step for bumping the opened PRs assigned to the milestone, we already notified the authors in the pre release section but I think it would be helpful to explicitly add this step, wdyt?

**Example:**
```
- [ ] [Bump open Gutenberg Mobile PRs assigned to {{gb_mobile_version}} milestone]({{milestone_url}}) to next milestone.
```